src/server/utils/getCashaySchema.js

```diff
@@ -3,4 +3,6 @@ require('babel-polyfill');
 const {transformSchema} = require('cashay');
 const graphql = require('graphql').graphql;
 const rootSchema = require('../graphql/rootSchema');
+const r = require('../database/rethinkDriver');
```

Comment:
I ran into trouble here when using `npm run dev`. Each time the piping watch ran, it would run `cashay-loader`, evaluate this `getCashaySchema.js` and then drain the rethink pool. Made for much sadness. We'll probably have to introduce a webpack flag/contant. 😩